### PR TITLE
Fix error with last version (Compat 241)

### PR DESCRIPTION
**Pull request by d3v2a**
- Migrate to new plugin structure
 - Fix error with ActionUpdateThread
 - Add icon from https://www.svgrepo.com

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.jar
Comment (silvasur) [inline gradle/wrapper/gradle-wrapper.jar:1]: How was this file generated? I tried to run `gradle wrapper` with gradle version 8.6 and got a different JAR in the end? How can I verify that this file is legit?

File: build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+version = "0.8.0"
+
+repositories {
+    mavenCentral()
+}
+
+// Configure Gradle IntelliJ Plugin
+// Read more: https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html
+intellij {
+    version.set("2023.2.5")
```
Comment (silvasur) [inline build.gradle.kts]: Hmm, is there a way to have a lower version? I'd like to have an as broad as possible compatibility. 2022.3﻿ seems to be the version when `getActionUpdateThread` was introduced and I was able to build with that version (and also setting `sinceBuild.set` to 223)

File: src/main/resources/META-INF/pluginIcon.svg
Comment (silvasur) [inline src/main/resources/META-INF/pluginIcon.svg:1]: What's the license for this file? Can this just be imported into this repo without any legal issues? If in doubt, I'd prefer to just stay without an icon, it's not really needed for the plugin.

File: build.gradle.kts [hunk 2]
```diff
@@ -32,6 +32,7 @@ tasks {
 
     patchPluginXml {
         sinceBuild.set("232")
+        untilBuild.set("500.*")
```
Comment (silvasur) [inline build.gradle.kts:35]: Why is this needed?

**Review by silvasur — CHANGES_REQUESTED** [review]
Thanks for the pull request, this fixes the deprecation error. Please check the comments for the individual files.

File: gradle/wrapper/gradle-wrapper.jar
Comment (d3v2a) [reply gradle/wrapper/gradle-wrapper.jar:1]: Just create a new plugin with idea and do a diff, it was generated by the ide

File: build.gradle.kts (hunk 2 above)
Comment (d3v2a) [reply build.gradle.kts:35]: This is the max version compatible and is required.
Surely to force the maintainers to update more frequently plugin.
I put this, but I don't know if the marketplace accepts this

File: src/main/resources/META-INF/pluginIcon.svg
Comment (d3v2a) [reply src/main/resources/META-INF/pluginIcon.svg:1]: The License is in src/main/resources/META-INF/plugin.xml line 38.

File: build.gradle.kts (hunk 1 above)
Comment (d3v2a) [reply build.gradle.kts]: Added in last commit

File: gradle/wrapper/gradle-wrapper.jar
Comment (silvasur) [reply gradle/wrapper/gradle-wrapper.jar:1]: Ah I see, confirmed :)

File: build.gradle.kts (hunk 1 above)
Comment (silvasur) [reply build.gradle.kts]: Thanks!

File: src/main/resources/META-INF/pluginIcon.svg
Comment (silvasur) [reply src/main/resources/META-INF/pluginIcon.svg:1]: Ah okay, I checked svgrepo now and apparently that short line is really all that is necessary. Then this is OK :)

File: build.gradle.kts (hunk 2 above)
Comment (silvasur) [reply build.gradle.kts:35]: Okay, I see. If the current versioning scheme is kept, we're good until 2050, so that's fine by me :).

**Review by silvasur — APPROVED** [review]
Looks good to me now, will merge it. Thank you for your merge request! If you'd like, I'll add a `CONTRIBUTORS` file to credit you a bit more explicitly than just in the git commit authorship data. If you'd like that, please reply to me, which name and URL (if you want) I should put there :)

**Comment by silvasur** [comment]
I've now merged it and submitted the new plugin version to the jetbrains marketplace, might take a while for it to get approved.

**Pull request merged**